docs/docs-beta/docs/guides/asset-checks.md

```diff
@@ -1,64 +1,74 @@
 ---
-title: "Test assets with Asset Checks"
+title: "Testing assets with Asset Checks"
 sidebar_position: 10
+sidebar_label: "Asset Checks"
 ---
 
-Asset checks in Dagster provide a way to define and execute different types of data quality checks on your data assets directly in Dagster.
+Asset checks in Dagster provide a way to define and execute different types of data quality checks on your data assets.
 
 This guide covers the most common use cases for testing assets with asset checks and taking action based on the result.
 
 <details>
 <summary>Prerequisites</summary>
+
+To follow this guide, you'll need:
+
 - Familiarity with [Assets](/concepts/assets)
 </details>
 
-## Testing assets with a single asset check
+## Defining a single asset check \{#single-check}
+
+:::tip
+Dagster's dbt integration can model existing dbt tests as asset checks. Refer to the [dagster-dbt documentaiton](/integrations/dbt) for more information.
+:::
 
-The example below defines a single asset check on an asset that fails if the `order_id` column of the asset contains a null value.
+Single asset checks are defined using the `@asset_check` decorator.
```

Comment:
We havent really defined what a 'single' asset check is

Reply:
Added something about this to the page intro.